cmd/topology/topology.go

```diff
@@ -44,8 +60,64 @@ var (
 		Short: "push or generate certs for nodes in topology",
 		RunE:  certFn,
 	}
+	resetCfgCmd = &cobra.Command{
+		Use:   "reset -skip <topology> <device>",
+		Short: "reset configuration of device (if device not provide reset all nodes)",
+		RunE:  resetCfgFn,
+	}
+)
+
+var (
+	skipReset bool
 )
 
+func resetCfgFn(cmd *cobra.Command, args []string) error {
+	if len(args) != 1 {
+		return fmt.Errorf("%s: missing args", cmd.Use)
+	}
+	topopb, err := topo.Load(args[0])
+	if err != nil {
+		return fmt.Errorf("%s: %w", cmd.Use, err)
+	}
+	s, err := cmd.Flags().GetString("kubecfg")
+	if err != nil {
+		return err
+	}
+	t, err := topo.New(s, topopb)
+	if err != nil {
+		return fmt.Errorf("%s: %w", cmd.Use, err)
+	}
+	ctx := cmd.Context()
+	t.Load(ctx)
+	var nodes []*node.Node
+	if len(args) == 1 {
+		nodes = t.Nodes()
+	} else {
+		n, err := t.Node(args[1])
+		if err != nil {
+			return err
+		}
+		nodes = append(nodes, n)
+	}
+	var resetable []node.Reseter
```

Comment:
nit: resettable and later in the interface its resetter

Reply:
fixed